code_challenges/code_wars/increment_string.py

```diff
@@ -9,14 +9,14 @@ def increment_string(characters):
 
     parts = re.match(r"([a-zA-Z0-9]+[a-zA-Z])([0-9]+)", characters)
 
-    first, second = parts.groups()
+    first_half, nums_at_end = parts.groups()
 
-    characters = first
+    characters = first_half
```

Comment:
Part of the issue is this reassignment only to get a better variable name.
```python
characters = first_half
```
The reason I suggested calling it characters from the jump on line 12 was to avoid this needless reassignment.

Reply:
i see, i guess the reason i did it this way was readability to me. i understand that the reassignment isnt needed but i like that it shows that the string that is passed into the function is getting split up. is doing it this way frowned upon?

Reply:
It is frowned upon.  You can always pick a name that is more descriptive like `characters_first_half`